### DISCUSSION TITLE
How about a ACJ Version ???

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#2733] [Ideas]
Hello community! 
To my mind, the A32NX is by far the most useable plane in FS2020 by now. But we don´t have any long-haul Planes (the 787/747 are not useable with 75% inop systems...so, not interessting forenthusiasts)

What are you thinking about a A320NEO-ACJ Version of this great A32NX-MOD? 

https://www.acj.airbus.com/en/exclusive-products/acj320neo.html

It would be so awesome to serve VVIP long haul flight with such a nice plane!
_"The ACJ320neo can fly twenty-five passengers 6,000 nm/ 11,112 km or more than 13 hours non-stop."_
I´m not a programmer, only a real world A319/A320/A321 Pilot, but to my mind, it´s would be not so complicated to create this beauty. In real life, it could carry 3000GAL more fuel compared to the "stock-A320Neo"

My two questions to you guys here:
- is there anybody, who are also highly interessted in this one?
- is it a huge amount of  work to create a version with a extended Center-Tank (add 3000GAL) ? (IRL there are up to 5 additional center tanks)

Thank you for your answers and I hope we can cross the atlantic sometimes with a A32NX-ACJ :)

**Comment by GACSavannah** [comment]
I would LOVE to see it. Nice to see an A319 as well.
